host-configs/docker/docker-linux-ubuntu20.04-ivybridge-clang@10.0.0.cmake

```diff
@@ -27,9 +27,9 @@ else()
 
 endif()
 
-set(CMAKE_C_FLAGS "-fPIC -fPIC" CACHE STRING "")
+set(CMAKE_C_FLAGS "-fPIC -pthread -fPIC -pthread" CACHE STRING "")
```

Comment:
why is this being doubled up?

Reply:
https://github.com/spack/spack/blob/b8512983d999575d372e7ecaefa76896593bc94e/lib/spack/spack/build_systems/cached_cmake.py#L101

I think maybe we should be using `cxxflags` instead of `cppflags` as the latter might actually be referring to flags intended for the preprocessor?